students/uss_tomasz/lesson_11_argparse/11_1_regex_search/test_regex_search.py

```diff
@@ -0,0 +1,36 @@
+import os
+import unittest
+from regex_search import parse_args, get_file_names_list, find_regex_in_txt_files
+
+
+class TestRegexSearch(unittest.TestCase):
+    def setUp(self):
```

Comment:
As above - tests should not be executed in current directory. This could be used with unittest
https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory